pkg/agent/controller/networkpolicy/priority.go

```diff
@@ -324,7 +338,7 @@ func (pa *priorityAssigner) insertConsecutivePriorities(priorities types.ByPrior
 		upperBound = pa.priorityMap[pa.sortedPriorities[insertionIdx]]
 	}
 	// not enough space to insert Priorities.
-	if upperBound-lowerBound-1 <= uint16(numPriorities) {
+	if upperBound-lowerBound-1 < uint16(numPriorities) {
```

Comment:
you may want to add a unit test for this case if possible (when we have just enough space to fit the requested number of priorities), since as you mention in the PR description this was basically a bug

Reply:
Done.